java/performance/external/binaries-list

```diff
@@ -14,4 +14,4 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-EE416CB5D6AA88473EFE487F42DC1410360F7948 javacc-3.2.jar
+3314CFC8BBD82725013EA997506EE5C525E8153A javacc-7.0.5.jar
```

Comment:
Could it be possible to use the javacc-7.05 like this 
3314CFC8BBD82725013EA997506EE5C525E8153A net.java.dev.javacc:javacc:7.0.5

Reply:
Yes and that would be the right way. We try to drop usage of the binary server to the cases where it is actually necessary, a standard artifact from maven central should be preferred.

Reply:
I fix it, thanks for the observation.